src/physicalQuantity.ts

```diff
@@ -4,4 +4,8 @@ export class PhysicalQuantity {
   constructor(quantityType: string) {
     this.type = quantityType
   }
+
+   getNumber(unit: string): number {
```

Comment:
Would an abstract class be better here?
Force the implementor of the subclass to implement this method.
This solution would still result in the failure only being detected at runtime